shapely/tests/test_strtree.py

```diff
@@ -78,19 +72,6 @@ def test_init_with_invalid_geometry():
         STRtree(["Not a geometry"])
 
 
-def test_init_increases_refcount():
```

Comment:
The ref count tests and asserts no longer applicable now that we copy geometry array input to the tree